Invalid values: Argument: type, Given: "OpenAPI", Choices: "openapi", "har", "postman"

This problem drove me insane:

```
Run FILE_ID=$(nexploit-cli archive:upload --type openapi --token *** openapi.json)
nexploit-cli archive:upload [options] <file>

Uploads a archive to NexPloit.

Positionals:
  file  A collection your app's http/websockets logs into HAR file. Usually you can use browser dev tools or our browser web extension  [string] [required]

OAS Options
  -H, --header  A list of specific headers that should be included into request.  [array] [default: []]

Postman Options
  -H, --header    A list of specific headers that should be included into request.  [array] [default: []]
  -V, --variable  A list of specific variables that should be included into request. Only for Postman  [array] [default: []]

Options:
      --config     Path to the file with configuration
      --log-level  What level of logs to report. Any logs of a higher level than the setting are shown.  [choices: 0, 1, 2, 3, 4, "silent", "error", "warn", "notice", "verbose"] [default: 3]
      --api        NexPloit base URL  [required] [default: "https://nexploit.app/"]
      --insecure   Allows CLI to proceed and operate even for server connections otherwise considered insecure.  [boolean] [default: false]
      --proxy      SOCKS4 or SOCKS5 URL to proxy all traffic
  -t, --token      NexPloit API-key  [required]
  -T, --type       The specification type  [required] [choices: "openapi", "har", "postman"] [default: "har"]
  -d, --discard    Indicates if archive should be remove or not after scan running. Enabled by default.  [boolean] [default: true]
  -v, --version    Show version number  [boolean]
  -h, --help       Show help  [boolean]

Invalid values:
  Argument: type, Given: "OpenAPI", Choices: "openapi", "har", "postman"
Error: Process completed with exit code 1.
```
I've just run the same GitHub Action with `@neuralegion/nexploit-cli@7.14.0` and the file got pushed correctly.

--- a/src/Commands/UploadArchive.ts
+++ b/src/Commands/UploadArchive.ts
@@ -34,10 +34,7 @@ export class UploadArchive implements CommandModule {
           SpecType.POSTMAN
         ].map((x: string) => x.toLowerCase()),
         default: SpecType.HAR.toLowerCase(),
-        demandOption: true,
-        coerce(arg: string): SpecType {
-          return Helpers.selectEnumValue(SpecType, arg) as SpecType;
-        }
+        demandOption: true
       })
       .option('discard', {
         alias: 'd',
@@ -110,7 +107,9 @@ export class UploadArchive implements CommandModule {
       const parserFactory: ParserFactory = container.resolve(ParserFactory);
       const archives: Archives = container.resolve(Archives);
 
-      const parser: Parser = parserFactory.create(args.type as SpecType);
+      const parser: Parser = parserFactory.create(
+        Helpers.selectEnumValue(SpecType, args.type as string) as SpecType
+      );
       const file = await parser.parse(args.file as string);
 
       const spec: Spec = {

